tf_encrypted/keras/layers/convolutional.py

```diff
@@ -438,16 +475,24 @@ def compute_output_shape(self, input_shape):
     return [n_x, n_filters, h_out, w_out]
 
   def get_mask(self, in_channels):
-    mask = np.zeros((self.kernel_size[0],
-                     self.kernel_size[1],
-                     in_channels, in_channels * self.depth_multiplier))
+    """TODO"""
+    mask = np.zeros(
+        (
+            self.kernel_size[0],
+            self.kernel_size[1],
+            in_channels,
+            in_channels * self.depth_multiplier
+        )
+    )
     for d in range(self.depth_multiplier):
       for i in range(in_channels):
         mask[:, :, i, i + (d * in_channels)] = 1.
     return np.transpose(mask, [0, 1, 3, 2])
 
   def set_weights(self, weights, sess=None):
-    """ Sets the weights of the layer.
+    """
+    Sets the weights of the layer.
```

Comment:
Is this recommended? I seem to recall Google docstrings having the first (imperative) sentence inline with the opening of the quotation

Reply:
yeah, docstrings need a do-over at some point; luckily flake8 seems to capture them (but currently ignored in `setup.cfg`)